### PR TITLE
:mortar_board: Assignment tsp --- Implementation section 

### PR DESCRIPTION
**Pull request by jameshughes89**
### What

Describe implementing the GA for TSP. Also explain the additional things they can do for more points. 

### Testing
 :+1:

### CODE REVIEW

File: site/assignments/tsp.rst [hunk 1]
```diff
@@ -110,18 +110,78 @@ Python provides a simple way to load JSON files directly into dictionaries.
...
+Having the GA generate a visualization of results will provide an additional **2** points. How this is done is up to
+each individual, but ensure it is interesting, effective, clear, and well presented to ensure the points are awarded.
+The more creative the better.
+
+For example, having a final static visualization of a route is *fine*, but not particularly interesting. It would be
+better if the GA periodically presents the best path or a set of paths from the population. If planning on doing this in
+real time, it is not recommended to have this happen for each generation as it would take a very long time. Instead,
+perhaps only show the visualization every so often or save images to create an animation after evolution is done.
+
+Do not feel that visualizing the routs is the only option. It could be a visualization of some mechanism within the GA,
```
Comment (twentylemon) [inline site/assignments/tsp.rst:167]: routs

File: site/assignments/tsp.rst [hunk 2]
```diff
@@ -110,18 +110,78 @@ Python provides a simple way to load JSON files directly into dictionaries.
...
+
 
 Vehicle Routing Problem
 -----------------------
 
+If a GA is also applied to the *vehicle routing problem*, **5** additional points may be obtained. Be sure to have a
+working TSP implementation before attempting this more complex problem. Points will only be awarded for a complete and
+working implementation of a GA for the vehicle routing problem
+
+The vehicle routing problem is, to describe it briefly, a generalization of TSP that is multi-objective (more than one
```
Comment (twentylemon) [inline site/assignments/tsp.rst:178]: Where is this problem described with detail? Is the expectation that the students look it up?

**Review by twentylemon — APPROVED** [review]

File: site/assignments/tsp.rst (hunk 2 above)
Comment (jameshughes89) [reply site/assignments/tsp.rst:178]: yeeeee

Comment (jameshughes89) [reply site/assignments/tsp.rst:178]: (I'm adding this fact to the description).

**Pull request merged**